modules/007-registrypackages/images/iptables/scripts/install

```diff
@@ -17,34 +17,74 @@ mkdir -p /opt/deckhouse/bin
 cp -f xtables-legacy-multi /opt/deckhouse/bin
 cp -f xtables-nft-multi /opt/deckhouse/bin
 
-kubeletChainsRegex='^:(KUBE-IPTABLES-HINT|KUBE-KUBELET-CANARY)'
-NFT_KERNEL_LIMIT=3.14 # nf_tables_ipv6, nf_tables_bridge, nf_tables_arp allow since Linux kernel >= 3.14.
-CURRENT_KERNEL_VERSION=$(uname -r | awk -F"-" '{print $1}') #version kernel view format: a.b.c-generic to a.b.c.
-IPTABLES_LEGACY_RULE=$( (/opt/deckhouse/bin/xtables-legacy-multi iptables-save || true ) 2>/dev/null | grep  -E ${kubeletChainsRegex} | wc -l )
+if [ -f /opt/deckhouse/bin/iptables ]; then
+  # if iptables symlinks already exist, we doesn't update it
+  exit 0
+fi
 
-function check_python() {
-  for pybin in python3 python2 python; do
-    if command -v "$pybin" >/dev/null 2>&1; then
-      python_binary="$pybin"
-      return 0
+function kernelNFTSupport() {
+  local configNFTablesRegex='CONFIG_NF_(TABLES|TABLES_IPV4)=(y|m)'
+  local configFiles=(
+    /boot/config-$(uname -r)
+    /proc/config.gz
+    /boot/config
```

Comment:
`/boot/config` isn't safe imho. But `/boot/config-$(uname -r)` is.

Reply:
done